<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RBTree</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow {
background-color: #131313;
color: #ffffff;
border-radius: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>10</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #393939;
color: white;
border-radius: 5px;</string>
    </property>
    <property name="title">
     <string>Элемент дерева</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(38, 38, 38);
border: none;</string>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Введите значение</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insertButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#insertButton{
	background-color: rgb(38, 38, 38);
	border: none;
}

#insertButton:hover{
	border: 1px solid #393939
}

#insertButton:pressed{
	background-color: #393939;
}</string>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="insertRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="removeRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>84</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;</string>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #131313;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="text">
      <string>Элемент дерева</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>lineEdit</zorder>
    <zorder>insertButton</zorder>
    <zorder>insertRadioButton</zorder>
    <zorder>removeRadioButton</zorder>
    <zorder>label_2</zorder>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>701</y>
      <width>1261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#horizontalScrollBar
{
color: green;
/*color: rgb(38, 38, 38);*/
/*margin: 0px 0px 0px 0px;*/
/*scrollbar-color: #262626 #131313*/
}</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>0</y>
      <width>21</width>
      <height>701</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
